examples/doc-scan/src/main/resources/application.yml.example

```diff
@@ -0,0 +1,15 @@
+server:
+  port: 8443
+  ssl:
+    enabled: true
+    key-store: "server.keystore.jks"
+    key-store-password: javaexample
+  servlet:
+    session:
+      persistent: true
+      timeout: 360000
+com:
+  yoti:
+    docscan:
+      sdkId: "your-sdk-id-from-yoti-hub"
+      accessSecurityKey: "your-pem-file-from-yoti-hub"
```

Comment:
This mirrors the same key in the other spring example project, so I'd rather keep them the same for now, and update them both in the major

Reply:
We aren't going to have many clients using both this and the profile example are we? I think it might be better to make the change to this now, so that it won't change for clients on this one if they are using it (don't know if we consider changes to the example projects as breaking changes anyway)